src/main/java/seedu/address/model/meeting/MeetingDuration.java

```diff
@@ -11,9 +11,10 @@
  */
 public class MeetingDuration {
     public static final String MESSAGE_CONSTRAINTS =
-            "Duration should be a decimal number given in hours and range from 0 < duration <= 24";
-
-    public final float duration;
+            "Duration should be a decimal number with 4 decimal places "
```

Comment:
```suggestion
            "Duration should be a decimal number (4dp) "
```
I think this can be shortened